src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
Then let's remove the `opt_callback`, and do `callback(true)` with the rest of the attempt change size stuff.